packaging/generic-rpm/amazon-ecs-init.spec

```diff
@@ -16,6 +16,8 @@
 %global _cachedir %{_localstatedir}/cache
 %global bundled_agent_version %{version}
 %global no_exec_perm 644
+%global _missing_build_ids_terminate_build 0
```

Comment:
for my understanding, what is this used for?

Reply:
I encountered following error when running `make rpm` on fedora 34.
```
RPM build errors:
    Missing build-id in /home/fedora/go/src/github.com/aws/amazon-ecs-init/BUILDROOT/ecs-init-1.57.0-1.fc33.x86_64/usr/libexec/amazon-ecs-volume-plugin
    Generating build-id links failed
```
This seems to be a new thing on fedora 34. Later realized I was running wrong target but forgot to delete this line. I have tried to remove this line and the build passed. I can remove this from generic-rpm spec. Do we want to add this to rpm spec?

Reply:
i'm not sure if it's a good idea to let the build pass if the build-id is missing (which seems to be what this macro does). unless you are certain that missing this won't be an issue, i'd suggesting removing it